### DISCUSSION TITLE
Normal / Full format Ventoy USB

### DISCUSSION THREAD

**Discussion by sj631** [ventoy/Ventoy#901] [Q&A]
How can we normal/full format a USB before installing ventoy on it?

**Comment by maxmod** [comment]
Use https://formatusb.com/download it delete all partition etc also remove  i am also using.

**Comment by chromer030** [comment]
If you are using Linux you can use Gparted or KDE Partition Editor to fully format whole disk , change or recreate partition table ,resize partitions and change label and name of partition.
On macOS you can use Disk Utility for these purpose.
and on Windows you can use free version of MiniTool Partition Wizard , or you can use command-line diskpart.
